plot_sweep.py

```diff
@@ -20,36 +20,33 @@
 # from collections import defaultdict
 from datetime import datetime as dt
 from torchbenchmark.util.data import load_data_dir, load_data_files
-from torchbenchmark.score.compute_score import compute_score
-from torchbenchmark.score.generate_score_config import generate_bench_cfg
-
+from torchbenchmark.score.compute_score import TorchBenchScore
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument("data_dir", nargs='+', 
-                        help="One or more directories containing benchmark json files. " 
+    parser.add_argument("data_dir", nargs='+',
+                        help="One or more directories containing benchmark json files. "
                              "Each directory will be plotted as a separate series. "
                              "By default, the first file in the first directory will be used"
                              " to generate a score configuration with a target of 1000,"
                              " and everything else will be relative to that.")
     parser.add_argument("--output_html", default='plot.html', help="html file to write")
-    parser.add_argument("--score_heirarchy", default='torchbenchmark/score/score.yml', 
+    parser.add_argument("--score_heirarchy", default='torchbenchmark/score/score.yml',
```

Comment:
Maybe use "compute_score.SPEC_FILE_DEFAULT" to replace the default value here?

Reply:
Yeah, I can change that.